modules/videoio/cmake/detect_xine.cmake

```diff
@@ -0,0 +1,12 @@
+if(NOT HAVE_XINE AND PKG_CONFIG_FOUND)
+  pkg_check_modules(XINE libxine QUIET)
+  if(XINE_FOUND)
+    set(HAVE_XINE TRUE)
+  endif()
+endif()
+
+if(HAVE_XINE)
+  add_3p_target(xine "${XINE_INCLUDE_DIRS}" "${XINE_LIBRARIES}" "HAVE_XINE")
+endif()
+
+set(HAVE_XINE ${HAVE_XINE} PARENT_SCOPE)
```

Comment:
> PARENT_SCOPE

It is better to remove that.

Unnecessary "function" scope should be removed (perhaps it is already done [here](https://github.com/opencv/opencv/pull/12541/files#diff-a8bb897ccface5bd5e7eff5548b73f77R299)).

Reply:
It is not about "status" text only.
It is about the similar process for all external dependencies, including zlib (which is used for opencv_core directly, and for libtiff dependency) and imgcodecs dependencies.